api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/quartz/SchedulingProperties.java

```diff
@@ -30,6 +30,14 @@
      */
     private boolean enabled = true;
 
+    /**
+     * Overrides enabled property value of true if this property does not match hostname of CAS server.
```

Comment:
Use javadoc syntax when referring to `enabled`; makes updates much easier later on.